Source/UltraleapTrackingCore/Private/LeapUtility.cpp

```diff
@@ -184,4 +193,46 @@ void FLeapUtility::ConvertFStringArrayToCharArray(const TArray<FString>& FString
 void FLeapUtility::SetLastArrayElemNull(const char*** ConstCharArrayPtr, int32 LastIdx)
 {
 	(*ConstCharArrayPtr)[LastIdx] = NULL;
-}
\ No newline at end of file
+}
+
+FString FLeapUtility::GetAnalyticsData(size_t& Size)
+{
+	FAnalytics Analytics;
+	Analytics.telemetry.app_title = FApp::GetProjectName();
+
+#if WITH_EDITOR
+	Analytics.telemetry.app_type = "editor";
+#else
+	Analytics.telemetry.app_type = "build";
+#endif
+
+	Analytics.telemetry.engine_name = "Unreal";
+	FString UnrealVersion = FString::FromInt(ENGINE_MAJOR_VERSION);
+	UnrealVersion += ".";
+	UnrealVersion += FString::FromInt(ENGINE_MINOR_VERSION);
+	Analytics.telemetry.engine_version = UnrealVersion;
+	Analytics.telemetry.installation_source = "github";
```

Comment:
I guess there's no way of telling if it came from the marketplace?

Reply:
There's, at line 226 "Analytics.telemetry.installation_source = "EpicGamesLauncher";"